eacl/target.go

```diff
@@ -45,6 +47,7 @@ func (t Target) CopyTo(dst *Target) {
 //
 // The value returned shares memory with the structure itself, so changing it can lead to data corruption.
 // Make a copy if you need to change it.
+// Deprecated: use [Target.Accounts] instead.
 func (t *Target) BinaryKeys() [][]byte {
 	return t.keys
```

Comment:
implementation of this method must be changed for backward compatibility. Some `t.keys` items can be 25B now, so they must be excluded